### PR TITLE
Fix NotificationWindow Feature

### PR DESCRIPTION
**Pull request by willjustin**
Most full-screen apps can display NotificcationWindow , and the NotificcationWindow background is transparent.
Lenovo_Legion_Toolkit_Lrzy4h3Ztn

### CODE REVIEW

**Review by BartoszCichecki — CHANGES_REQUESTED** [review]
What is the point of these changes? Which issue is this fixing?

File: LenovoLegionToolkit.WPF/Windows/Utils/NotificationWindow.cs [hunk 1]
```diff
@@ -37,8 +37,8 @@ public class NotificationWindow : UiWindow
 
     private readonly Label _textBlock = new()
     {
-        FontSize = 16,
-        FontWeight = FontWeights.Medium,
+        FontSize = 18,
+        FontWeight = FontWeights.UltraBold,
```
[the remark this answers is not in the document]
Comment (willjustin) [reply LenovoLegionToolkit.WPF/Windows/Utils/NotificationWindow.cs:41]: After the NotificationWindow becomes transparent, the text information is a little less obvious

File: LenovoLegionToolkit.WPF/Utils/NotificationsManager.cs [hunk 2]
```diff
@@ -49,7 +49,7 @@ private void OnNotificationReceived(Notification notification)
                 if (Log.Instance.IsTraceEnabled)
                     Log.Instance.Trace($"Some application is in fullscreen.");
 
-                return;
```
[the remark this answers is not in the document]
Comment (willjustin) [reply LenovoLegionToolkit.WPF/Utils/NotificationsManager.cs:52]: Well, giving the user an option is fine

[the remark this answers is not in the document]
Comment (willjustin) [reply LenovoLegionToolkit.WPF/Utils/NotificationsManager.cs:52]: So I'll add two options  in the Settings, one transparent, one full screen display, is that OK

[the remark this answers is not in the document]
Comment (willjustin) [reply LenovoLegionToolkit.WPF/Utils/NotificationsManager.cs:52]: I followed lenovo Hotkeys to prevent notifications from obscering the content below

**Pull request closed without merging**